### PR TITLE
sci-libs/libspatialindex: version bump to 1.9.3

### PR DESCRIPTION
**Pull request by expeditioneer**
Signed-off-by: Dennis Lamm <expeditioneer@gentoo.org>

### CODE REVIEW

File: sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_PN="spatialindex-src"
+MY_P="${MY_PN}-${PV}"
```
Comment (a17r) [inline sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild:9]: `MY_PN` is nowhere used except here, so you might as well just set `MY_P="spatialindex-src-${PV}"`

File: sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+DESCRIPTION="General framework for developing spatial indices"
+HOMEPAGE="https://libspatialindex.org/
+	https://github.com/libspatialindex/libspatialindex"
+SRC_URI="https://github.com/libspatialindex/${PN}/releases/download/${PV}/${MY_P}.tar.bz2"
+LICENSE="MIT"
+
+KEYWORDS="~amd64 ~x86"
```
Comment (a17r) [inline sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild:17]: `LICENSE` goes into the block below. Here's how this should look, in order:

```
LICENSE
SLOT
KEYWORDS
IUSE
```

File: sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+SLOT="0/6"
+IUSE="test"
```
Comment (a17r) [inline sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild:19]: This requires also adding `RESTRICT="!test? ( test )"`

File: sci-libs/libspatialindex/metadata.xml [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
+	<maintainer type="project">
+		<email>3dprint@gentoo.org</email>
+		<name>Gentoo 3D print</name>
+	</maintainer>
+	<longdescription>
+		Extensible framework for robust spatial indexing
+	</longdescription>
```
Comment (a17r) [inline sci-libs/libspatialindex/metadata.xml:10]: That's not very long for a `longdescription`. Drop this and rather improve `DESCRIPTION` in the ebuild?

File: sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild (hunk 1 above)
Comment (expeditioneer) [reply sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild]: DONE

File: sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild (hunk 3 above)
Comment (expeditioneer) [reply sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild:18]: DONE

File: sci-libs/libspatialindex/metadata.xml (hunk 4 above)
Comment (expeditioneer) [reply sci-libs/libspatialindex/metadata.xml]: OK, longdescription removed, improved description as desired

File: sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild (hunk 2 above)
Comment (expeditioneer) [reply sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild]: DONE

**Comment by expeditioneer** [comment]
@a17r No just if you are now fine with the ebuild itself so that i could merge this pr. Thank you for your review.

**Pull request closed without merging**